client/lib/terms/constants.js

```diff
@@ -13,3 +13,9 @@ module.exports.action = keyMirror( {
 	FETCH_TAGS: null,
 	SET_CATEGORY_SELECTED_ITEMS: null
 } );
+
+module.exports.defaultNonHierarchicalQuery = {
+	number: module.exports.MAX_TAGS,
+	order_by: 'count',
+	order: 'DESC'
+};
```

Comment:
Doesn't matter much here since I suspect this is a file we'll seek to eliminate altogether, but in general I don't think it's a big deal to mix import / export syntaxes. If anything, the eyesore will hasten follow-up 😄